### ISSUE TITLE
Review accessibility blockers and add to microsites stable release blocker list

### ISSUE THREAD

**Issue by finnlewis** [localgovdrupal/localgov_microsites#482]
@markconroy mentioned there are some accessibility issues that should be blockers to a stable release. 

https://github.com/localgovdrupal/localgov_microsites_base/issues?q=is%3Aissue+is%3Aopen+label%3Aaccessibility

Please could you add any issues to this meta issue: https://github.com/localgovdrupal/localgov_microsites/issues/378 ?

**Comment by markconroy** [comment]
Closing this. All our other issues are not blockers to a stable release.
